README.md

```diff
@@ -35,7 +35,7 @@ That should generate `fsatrace.exe`, `fsatracehelper.exe`,
 Make sure the .dll or .so files are in the same path as the `fsatrace`
 executable and run:
 
-    fsatrace <options> <output-file> -- <command>
+    fsatrace <options> <output-file> [-s buf_size] -- <command>
```

Comment:
I'd much prefer a flag style since it is cleaner. For example environment variable based flag passing will inherit global env, which can be surprising sometimes. WDYT?

Reply:
Global env shouldn't be a problem, nobody should have a `FSAT_BUFSZ` in their global env. 
My rationale for avoiding passing as arguments:

- I don't want to use an external library for argument parsing. At some point we might want to add some additional optional argument and it will get messy.
- My build tool is compiled and launching a build increasing the buffer size would imply recompiling the build tool or adding some way to change this parameter externally. I'd prefer to just go to the script that launches it and add `env FSAT_BUFSZ=XXX ./build`.

Reply:
I totally understand your concerns, and why you don't want to switch to CLI flags at the moment, but IMHO flags is still a cleaner interface, and should be preferred if possible.

I uploaded another commit to switch to environment variable. I added a log to print out the buffer size just in case the environment variable is set unexpectedly, should be helpful in debugging.

PTAL! Thanks.